# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/AudioConduit.h
# Commit: b99b71e755e4
# Full Hash: b99b71e755e470a5dd152a5d6313f3002b2b9ba9
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-12-18 09:43:08
# Description:
#   Bug 1744081 - Move VideoConduit::Shutdown asyncness ownership into VideoConduit. r=bryce,ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D133933
# ==============================================================================

diff -r f69c775a1024 -r b99b71e755e4 dom/media/webrtc/libwebrtcglue/AudioConduit.h
--- a/dom/media/webrtc/libwebrtcglue/AudioConduit.h	Fri Dec 17 20:29:15 2021 +0000
+++ b/dom/media/webrtc/libwebrtcglue/AudioConduit.h	Fri Dec 17 20:34:18 2021 +0000
@@ -131,7 +131,7 @@
 
   void DeliverPacket(rtc::CopyOnWriteBuffer packet, PacketType type) override;
 
-  void Shutdown() override;
+  RefPtr<GenericPromise> Shutdown() override;
 
   WebrtcAudioConduit(RefPtr<WebrtcCallWrapper> aCall,
                      nsCOMPtr<nsISerialEventTarget> aStsThread);